contrib/docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y wget unzip git bzip2
+RUN wget https://github.com/neuropoly/spinalcordtoolbox/archive/master.zip
+RUN unzip master.zip
+WORKDIR spinalcordtoolbox-master
```

Comment:
A comment on how run commands. Each time `RUN` is called, a new volume is created. What I suggest is to move `WORKDIR` above the `RUN` command. And merge all the run commands into 1 `RUN` command.

```
RUN apt-get update && \
apt-get install -y wget unzip git bzip2 && \
wget https://github.com/neuropoly/spinalcordtoolbox/archive/master.zip && \
unzip master.zip && \
...
```

Reply:
You mean layer ? Docker isn't going to create volumes every time `RUN` is called. When the `RUN` is executed, another layer is created on the top of the previous one. Since Docker uses Copy on Write (CoW), in this case, the final image size will be the same, so in my opinion, there will be no gains in merging all `RUN` commands together except for making it harder to read. I can merge them if you want, but I don't think it will be better though.

Reply:
Correction, you are right, it creates a new layer with each `run`. It is a question of best practice. But, it seams that it sounds like it is a duplicate of https://github.com/neuropoly/docker , am I wrong?

Reply:
I wasn't able to find an **Ubuntu** Dockerfile at https://github.com/neuropoly/docker, only a Debian image based on the Neurodebian. Let me know if you guys want me to close this PR without merging.

Reply:
no, i like the idea of having a container inside SCT repos, so i would keep this PR.
we need, however to associate a documentation with it, so that people know it exists and what to do with it.
this repos:  https://github.com/neuropoly/docker is made for the lab in general, not specific to SCT, so it might become messy at some point, without commit regulations, etc.